registry/application/services/organization_publisher_service.py

```diff
@@ -82,6 +84,7 @@ def update_organization(self, payload, action):
         updated_state = Organization.next_state(current_organization, updated_organization, action)
         if updated_state == OrganizationStatus.ONBOARDING.value:
             self.notify_approval_team(updated_organization.id, updated_organization.name)
+            self.notify_user_on_start_of_onboarding_process(updated_organization.id, recipients=[self.username])
```

Comment:
self.username will only mail the current logged in user. it can be member or the owner. can we send mail to specific owner only

Reply:
@pratik-vii marking as resolved. As in this case, the user will be the owner only.